src/sagas.js

```diff
@@ -436,27 +492,22 @@ function* startProgress ({ vmId, poolId, name }) {
   }
 }
 
-function* getSingleInstance ({ vmId, poolId }) {
-  const fetches = [ fetchSingleVm(getSingleVm({ vmId })) ]
-  if (poolId) {
-    fetches.push(fetchSinglePool(getSinglePool({ poolId })))
-  }
-  yield all(fetches)
-}
-
 function* stopProgress ({ vmId, poolId, name, result }) {
```

Comment:
obsolete

Reply:
I will leave `startProgress` and `stopProgress` in for now since they're used only for VM actions and the `VmActions` toolbar buttons use that state indicator as part of their enabled state calculations.